<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyItemWidget</class>
 <widget class="QWidget" name="MyItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
	background:#CCC;
}

#titleBarWidget{
	background:#888;
	min-height:40px;
	max-height:40px;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

#titleBarWidget QPushButton{
	min-width:20px;
	max-width:20px;
	min-height:20px;
	max-height:20px;
	padding: 0;
	color: #CCC;
	background: #333;
	border: none;
	border-radius: 10px;

}
#lbl_title{
	font-size:14px
}
</string>
  </property>
  <widget class="QWidget" name="titleBarWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>标题</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>1</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget{
	background:#CCC;
}

#titleBarWidget{
	background:
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>262</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>195</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_itemName">
       <property name="text">
        <string>Item名称:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_ItemName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_itemMyType">
       <property name="text">
        <string>Item类别:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_itemMyType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Test1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Test2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_ItemToggle">
       <property name="text">
        <string>Item注释:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_ItemToggle"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_itemPicture">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
